ban/tests/test_models.py

```diff
@@ -537,3 +537,79 @@ def test_position_center_coerce(given, expected):
         assert center.coords == expected
     else:
         assert not center
+
+
+def test_municipality_select_use_default_orderby():
```

Comment:
Those tests should be in the http modules, given we'll only use the order_by when building a collection for the API.